### PR TITLE
[Bug] LPS-158311 Clean up background task when master node is re-elected and send notifications

### PR DESCRIPTION
**Pull request by dantewang**
https://issues.liferay.com/browse/LPS-158311

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:clustering

**Comment by dantewang** [comment]
@zhangyan0701 Could you review the test results when they finish? Thanks!

**Comment by zhangyan0701** [comment]
ci:test:acceptance

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/portal-background-task/portal-background-task-service/src/main/java/com/liferay/portal/background/task/internal/notification/BackgroundTaskFailedUserNotificationHandler.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+		JSONObject jsonObject = JSONFactoryUtil.createJSONObject(
+			userNotificationEvent.getPayload());
+
+		return StringUtil.replace(
+			getBodyTemplate(), new String[] {"[$TITLE$]", "[$BODY$]"},
+			new String[] {
+				_language.get(serviceContext.getLocale(), "background-task"),
+				_language.format(
+					serviceContext.getLocale(), "background-task-has-failed",
```
Comment (tinatian) [inline modules/apps/portal-background-task/portal-background-task-service/src/main/java/com/liferay/portal/background/task/internal/notification/BackgroundTaskFailedUserNotificationHandler.java:57]: @dantewang Please add this to the message "please retrigger the failed background task if needed"

Comment (dantewang) [reply modules/apps/portal-background-task/portal-background-task-service/src/main/java/com/liferay/portal/background/task/internal/notification/BackgroundTaskFailedUserNotificationHandler.java:57]: @tinatian See https://github.com/liferay-core-infra/liferay-portal/pull/1212/commits/ec201a5576aa3adfd582165b7b2f2f5aa4c143c9#diff-ef119629946c9b5876f3ddaf7e739fa3316781f38535fa10141db62509a5d684R2219

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/121230

**Pull request closed without merging**